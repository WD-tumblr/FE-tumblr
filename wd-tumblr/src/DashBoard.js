import React from 'react';
import Modal from './components/Modal'
const DashBoard = props => {
  return <div>
    DashBoard
    <Modal show/>
    </div>;
};

export default DashBoard;
