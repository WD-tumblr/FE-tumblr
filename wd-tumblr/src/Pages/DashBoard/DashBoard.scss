.dashboard__container {
  padding-top: 92px;
  padding-left: 5px;
  padding-right: 5px;
  margin: 0 auto;
}

.postbar {
  display: flex;
  margin-bottom: 20px;
  & nav {
    background: #fff;
  }
}


.posts {
  display: flex;
}



.postbutton {
 & a {
   display: flex;
   flex-direction: column;
   align-items: center;
   width: 75px;
   height: 95px;
 }
 & i {
    font-family: Tumblr-Icons;
    width: 48px;
    height: 48px;
    display: inline-block;
    font-size: 48px;
  }
}

.icon__post-text {
  color: #444;
  
  &::before {
    content: '\EA77';
    display: inline-block;
    line-height: 1;
   
  }
}


.postform__container {
  margin-top: 146px;
  display: flex;
  position: absolute;
}

.postform {
  background: #fff;
  color: #444;
  width: 500px;
  margin-bottom: 20px;
  border-radius: 3px;
  padding: 20px;
  input {
    width: 100%;
    height: 48px;
    border: none;
  }
  textarea {
     width: 100%;
     border: none;
     overflow: auto;
     outline: none;
      resize: none; 
  }
}


.postform__header {
  display: flex;
  justify-content: space-between;
 
}

.form__toolbar {
  display: flex;
  justify-content: space-between;
}
