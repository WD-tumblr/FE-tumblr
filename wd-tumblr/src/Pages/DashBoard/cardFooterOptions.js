export const cardFooterOptions = [
  {
    buttonText: 'Edit',
    handlerKey: 'EDIT'
  },
  {
    buttonText: 'Delete',
    handlerKey: 'DELETE'
  }
];
