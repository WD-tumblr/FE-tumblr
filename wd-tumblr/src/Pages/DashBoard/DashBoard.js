import React, { Component } from 'react';
import Modal from '../../components/Modal'
import './DashBoard.scss';


class DashBoard extends Component {
  render() {
    return (
      <div className="dashboard__container">
      <section>
        <div className="postbar">
          <div className="avatar"></div>
          <nav>
            <ul class="postbuttons">
              <li>
                <a>
                  텍스트
                  <i className="icon__post-text"></i>
                </a>
              </li>
            </ul>
          </nav>
        </div>  
        <ul className="posts">
          <li>
            <div className="avatar"></div>
          </li>
          <li>
            <div className="avatar"></div>
          </li>
        </ul>      
      </section> 
         <Modal />
      </div>
    );
  }
}


export default DashBoard;
