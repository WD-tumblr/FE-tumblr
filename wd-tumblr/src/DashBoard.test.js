import React from 'react';
import { shallow } from 'enzyme';
import DashBoard from './DashBoard';
describe('DashBorad', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<DashBoard debug />);

    expect(component).toMatchSnapshot();
  });
});
