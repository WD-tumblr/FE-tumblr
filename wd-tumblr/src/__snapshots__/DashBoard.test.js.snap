// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DashBorad should render correctly in "debug" mode 1`] = `
<div>
  DashBoard 
</div>
`;
